options/options.c

```diff
@@ -182,7 +182,6 @@ static const m_option_t mp_vo_opt_list[] = {
                       ({"default", -1})),
     OPT_CHOICE_OR_INT("fs-screen", fsscreen_id, 0, 0, 32,
                       ({"all", -2}, {"current", -1})),
-    OPT_FLAG("fs-black-out-screens", fs_black_out_screens, 0),
```

Comment:
can i leave the msg for it as NULL or should i add something along the lines "no equivalent replacement"?